### DISCUSSION TITLE
Can Strapi Deepl Plugin be used in Lifecycle Functions?

### DISCUSSION THREAD

**Discussion by daffahilmyf** [Fekide/strapi-plugin-translate#283] [Q&A]
Hello everyone,

I'm currently working on the translation aspect of my Strapi project, and I've come across a situation where I need to utilize a lifecycle function to accomplish a specific task. Specifically, I'm interested in integrating the Strapi Deepl plugin into my lifecycle function for this purpose.

Here's a bit more detail about my scenario:

- I have a content type that I want to automatically translate using the Strapi Deepl plugin whenever it's created or updated.
- I believe using a lifecycle function might be the right approach for this, but I'm unsure about the exact steps to integrate the Deepl plugin within the lifecycle function.

My questions are:
1. Is it possible to use the Strapi Deepl plugin within a lifecycle function?
2. If yes, could someone guide me through the general steps or provide an example of how to achieve this?

I've looked through the Strapi documentation and the Deepl plugin documentation, but I couldn't find specific information about this integration. Any insights, code examples, or guidance would be greatly appreciated.

Thank you in advance for your help!

**Comment by mandrasch** [comment]
Hey, any luck doing that? #curious 

I read about it here as well
https://github.com/Fekide/strapi-plugin-translate/issues/284

Cheers!

**Reply by mandrasch** [reply]
Update: I tried it today, but I got stuck at auto-updating localized content 🤯 See:
https://discord.com/channels/811989166782021633/1409925483885494313/1409925483885494313

(Demo: https://github.com/mandrasch/strapi-v4-auto-translate)
